tests/performance/run.mjs

```diff
@@ -1127,6 +1112,112 @@ function execCommandAndGetFirstOutput(command) {
   });
 }
 
+/**
+ * Format the given report object into an readable HTML string.
+ * @param {Object} reportObj
+ * @returns {string}
+ */
+function formatHtmlReport(reportObj) {
+  let str = "<div>\n";
+  str += "  <p>\n";
+  if (reportObj.success) {
+    str += "    ✅ Automated performance checks have passed ";
+  } else {
+    str += "    ❌ Automated performance checks have failed ";
+  }
+  if (reportObj.commitSha && reportObj.baseBranch) {
+    str += `on commit <code>${reportObj.commitSha}</code> with the base branch <code>${reportObj.baseBranch}</code>`;
+  }
+  str += ".\n";
+  str += "  </p>\n";
+  str += "  <details>\n";
+  str += "    <summary>Details</summary>\n\n";
+  str += "    <h2>Performance tests 1st run output</h2>\n";
```

Comment:
`\n\n` make the line `<h2>Performance tests 1st run output</h2>` displayed as code

Reply:
You're right, and I think it's more of a combination of the indent + line break, but I don't get why Markdown would do this thing here...

I did two line breaks on purpose so we better see the h2 when opening the report in a text editor

Reply:
The second run output has the same problem...

I think I'll juste remove the indentation :/